parser.cpp

```diff
@@ -245,7 +245,7 @@ std::string cParser::readToken( bool ToLower, const char *Break ) {
     // launch child parser if include directive found.
     // NOTE: parameter collecting uses default set of token separators.
 	if( expandIncludes && token == "include" ) {
-        std::string includefile = readToken(ToLower); // nazwa pliku
+		std::string includefile = deserialize_random_set(*this); //readToken(ToLower); // nazwa pliku
```

Comment:
Don't do this, non-deterministic parsing is unacceptable. Handle it in the consumer.

Reply:
I don't get it - you don't like the feature (including randomly one of several files) or the way it is implemented?

Reply:
Random includes are now limited only to mmd file, which has nothing to do with core simulation nor scenery.